composer.json

```diff
@@ -22,14 +22,14 @@
     ],
     "homepage": "https://docs.sonata-project.org/projects/SonataIntlBundle",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/config": "^4.4 || ^5.4 || ^6.0",
```

Comment:
you can simplify the configuration files, same as was done for other bundles

Reply:
Can you give me an example ? Didn't find another bundle where it was simplified

Reply:
https://github.com/sonata-project/SonataBlockBundle/pull/1060/files

You can take a look at this one. The services file changed a bit.

Reply:
Thanks. Done. I also remove the parameter with the service classes ; it's useless, isn't it ?

Reply:
Yes! We couldn't remove it before because it is a BC break.